feature-labs/src/main/feature/feature.xml

```diff
@@ -18,11 +18,6 @@ limitations under the License.
 <features name="jclouds-labs-${project.version}" xmlns="http://karaf.apache.org/xmlns/features/v1.0.0">
     <repository>mvn:org.apache.jclouds.karaf/jclouds-karaf/${jclouds.version}/xml/features</repository>
 
-    <feature name='jclouds-labs-all-blobstore' description='jclouds-labs - all - macro feature to bundle all blobstores' version='${project.version}'>
-        <feature version='${project.version}'>jclouds-b2</feature>
-        <feature version='${project.version}'>jclouds-google-cloud-storage</feature>
-    </feature>
-
```

Comment:
I suggest we keep this empty. This will let projects depending on it to:
  * not break
  * not care when modules are added/removed